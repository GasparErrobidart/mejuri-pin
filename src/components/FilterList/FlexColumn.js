import styled              from 'styled-components'

const FlexColumn = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  flex-wrap:wrap;
  position:fixed;
  top:60px;
  left:0;
  width:100%;
  @media (min-width:768px){
    position:static;
    top:auto;
    left:auto;
    width:auto;
  }
`

export default FlexColumn
