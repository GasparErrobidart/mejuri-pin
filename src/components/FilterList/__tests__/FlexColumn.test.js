import '@testing-library/jest-dom'
import 'jest-styled-components'

import React                 from 'react'
import {ThemeProvider}       from 'styled-components'
import { cleanup }           from '@testing-library/react'
import renderer              from 'react-test-renderer'

import theme                 from '../../GlobalStyle/themes/mejuri-pin'

import FlexColumn   from '../FlexColumn'


afterEach(cleanup)

test('Renders "FlexColumn" component', () => {
  const testText        = "FlexColumn Test Text"
  const component       = renderer.create(
    <ThemeProvider theme={theme}>
      <FlexColumn>{testText}</FlexColumn>
    </ThemeProvider>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
