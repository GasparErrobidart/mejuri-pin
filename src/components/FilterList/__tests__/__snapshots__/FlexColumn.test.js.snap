// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "FlexColumn" component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  position: fixed;
  z-index: 1;
  top: 60px;
  left: 0;
  width: 100%;
  -webkit-transition: opacity 300ms ease-in;
  transition: opacity 300ms ease-in;
  opacity: 1.0;
}

@media (min-width:768px) {
  .c0 {
    position: static;
    top: auto;
    left: auto;
    width: auto;
  }
}

<div
  className="c0"
>
  FlexColumn Test Text
</div>
`;
